asn-compiler/src/generator/int.rs

```diff
@@ -244,7 +236,9 @@ impl Generator {
         let token_string = tokens.join(",");
 
         let derive_token_string = format!("#[derive({})]\n", token_string);
-        let derive_token_stream: TokenStream = derive_token_string.parse().unwrap();
+        let auto_serde = "#[cfg_attr(feature = \"serde_support\", derive(serde::Serialize, serde::Deserialize))]";
```

Comment:
The `serde_support` will be a feature of the target crate (where the specs are to be compiled). Is that understanding correct?

I believe one issue with this approach is it is not explicit to the one compiling the specs that they have to define this feature. Instead it's probably just easier to use the opt-ins from the command line. 

I am not too sure what exactly does this feature achieve that cannot be trivially achieved from the command line?

Reply:
Yes that was the idea.

The idea was to reduce the complexity of the CLI.

I'm closing it